index.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <link rel="stylesheet" href='styles.css'>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Challenge</title>
+  </head>
+  <body>
+    <h1>Adam Smith</h1>
+    <p id="bio">Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.<a href="https://www.linkedin.com">linkedin</a> Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.</p>
+    <h2>Education history</h2>
+    <div id="education">
+      <ol>
+        <li>Bilim innovation school</li>
+        <li>ENUniversity</li>
+        <li>Kokshetau University</li>
+      </ol>
+    </div>
+    <h2>Projects</h2>
+    <div id="projects"></div>
+    <h2>Social media</h2>
+    <div id="social">
+      <ul>
+        <li><a href='https://www.facebook.com' class="social-media-links">Facebook</a></li>
+        <li><a href='https://www.instagram.com' class="social-media-links">Instagram</a></li>
+        <li><a href='https://www.twitter.com' class="social-media-links">Twitter</a></li>
```

Comment:
- [ ] Good job for using [GitHub inline comments feature](https://docs.github.com/en/github/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/reviewing-proposed-changes-in-a-pull-request) and you phrased the comments politely for suggested changes,  But Kindly makes sure that ,you  includes an explanation, a "why", for each requested change.

Ex. "I think it would be better to use the main tag in the body part. because  It is a semantic and structural element introduced in HTML5. The element is also useful for screen readers to skip the navigation and other irrelevant content and jump directly to the core content of the documents".